### PR TITLE
ADR32 Extend Cross Sectoral Standards

### PR DESCRIPTION
**Pull request by raimundo-henriques**
We accidentally closed the previous PR for the ADR on cross sectoral standards, so we are now opening this new one. For previous discussions, refer to #69

### CODE REVIEW

**Comment by bhadley** [comment]
ADR now ready for PACT Tech Working Group review; endorsement confirmed from Green x Digital, TfS, and Catena-X. Community members requested to review, provide feedback, and (if applicable) raise any fundamental objections by May 29. Final decision on ADR is planned during June 5 PACT Tech Working Group.

File: decisions/log/0032-cross-sectoral-standards.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+[For full details see here](https://flat-dollar-c04.notion.site/Extend-the-Cross-Sectoral-Standards-List-to-be-more-comprehensive-and-extensible-ddf602f360e14168b2a300d71a38f672)
+
+## Proposal
+To address the above challenges, we (jointly with Catena-X, TfS, and Green x Digital) propose the following for evaluation by the PACT community:
+
+* A new attribute which includes a set of options. These options would include those listed in technical proposal below (exact list pending final alignment with community)
+* The set of options may be evolved without breaking backwards compatibility
+
+We have also considered the possibility of disclosing a cross sectoral standard not contained in the set of options explicitly covered by the Technical Specifications. We concluded that this possibility was suboptimal, for the following reasons:
+* It would afford more complexity to the Technical Specifications, requiring a structured data type similar to [`ProductOrSectorSpecificRule`](https://wbcsd.github.io/data-exchange-protocol/v2/#dt-productorsectorspecificrule)
```
Comment (JohnVonNeumann) [inline decisions/log/0032-cross-sectoral-standards.md:31]: wording: i think that "introduce" would be a better word than "afford", for non-english speakers I think it would come off too wordy

File: decisions/log/0032-cross-sectoral-standards.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        :: for version 2.1 of the PACT Framework
+        Not recommended but permitted*:
+        : PACT Framework v1
+        :: for version 1 of the PACT Framework
+        : PAS2050
+        :: for the Publicly Available Specification (PAS) 2050, "Specification for the assessment of the life cycle greenhouse gas emissions of goods and services"
+
+        Advisement:
+            The enumeration of standards above CAN evolve in future revisions. A host system MUST accept ProductFootprints from later revisions with `crossSectoralStandards` containing values that are not defined in this specification.
+            The set of standards listed include some standards which PACT does not recommend given they have become deprecated and/or outdated. However, with the goal of enabling transparency, PACT Technical Specifications nevertheless includes these as options.
```
Comment (JohnVonNeumann) [inline decisions/log/0032-cross-sectoral-standards.md:73]: Is there a more structured way to indicated standards that are listed but not recommended due to deprecation? I think it would make sense to not have them all be viewed at a data level as the same, basically i think that the deprecation side of things should be codified

Comment (bhadley) [reply decisions/log/0032-cross-sectoral-standards.md:73]: Hmm yes that's the challenge, as some are formally deprecated (like Framework v1) whereas others are generally not recommended but not officially deprecated (PAS 2050). Do you have a suggestion on how to codify this differently (without adding too much complexity... :)

File: decisions/log/0032-cross-sectoral-standards.md (hunk 1 above)
Comment (bhadley) [reply decisions/log/0032-cross-sectoral-standards.md]: Thanks! Fixed!

**Review by bhadley — APPROVED** [review]
LGTM! Let's merge this in (tech specs changes have already been made)
